### DISCUSSION TITLE
Auto theme's change theme button is unclear

### DISCUSSION THREAD

**Discussion by SarahG-579462** [pradyunsg/furo#794] [Potential Issue] [answered]
I noticed that the auto theme button, the #svg-sun-half, is quite hard to understand from a user perspective. It took me a while to discover that it changes the theme, and even then only because I know that it is an option for furo. 

Could the theme-toggle logo be made clearer on auto? My suggestion would be to have half the logo be the sun, and half the moon:
![svgexport-new-plain](https://github.com/pradyunsg/furo/assets/91751417/8676369a-2cec-440d-9338-b75790f4fb1e)

(feel free to use or modify this logo, btw. Edited the current icons with inkscape)

This discussion is being raised from a comment on a pull request to replace the sphinx-rtd-theme for furo: https://github.com/Ouranosinc/xclim/pull/1731#discussion_r1583230789

**Comment by pradyunsg** [comment, marked as answer]
I agree. Pretty sure past me didn't really like those icons either but used them because there wasn't anything else available/around.

You successfully nerd-sniped me with this and, about half a day later[^1], https://github.com/pradyunsg/furo/commit/81536d7747c0ab7c5afc8255209bf73d932bafa7 brings new icons that are _so much nicer_.

![Screenshot 2024-05-05 at 22 24 53](https://github.com/pradyunsg/furo/assets/3275593/bb8a71f4-79db-4ebd-867e-1952b42b91d9) ![Screenshot 2024-05-05 at 22 24 44](https://github.com/pradyunsg/furo/assets/3275593/c942c011-36c6-4ba3-8169-f96821caafc4)

[^1]: Hand-editting SVGs is a pain, and I now have a workflow for dealing with such things. 😄

**Reply by SarahG-579462** [reply]
Amazing! thanks
